frontend/src/metabase-types/api/card.ts

```diff
@@ -64,7 +64,25 @@ export interface PublicCard {
   dataset_query: PublicDatasetQuery;
 }
 
-export type CardDisplayType = string;
+export type CardDisplayType =
+  | "table"
+  | "bar"
+  | "line"
+  | "pie"
+  | "scalar"
+  | "row"
+  | "area"
+  | "combo"
+  | "pivot"
+  | "smartscalar"
+  | "gauge"
+  | "progress"
+  | "funnel"
+  | "object"
+  | "map"
+  | "scatter"
+  | "waterfall"
+  | (string & unknown);
```

Comment:
I don't think we allow custom chart type.

Reply:
<img width="1472" alt="image" src="https://github.com/user-attachments/assets/266e932e-2580-451e-8ac1-0d3fa93539f2">

vs

<img width="569" alt="Screenshot 2024-09-03 at 14 19 29" src="https://github.com/user-attachments/assets/e9511a67-8e17-4440-92f3-a8f51baf4623">

Reply:
Woah, that's weird. In Webstorm it gives me all of the types. I think I'll need to use VS Code more

Reply:
If we don't allow custom chart types, then we can get rid of the string thing (does it have a name?)

Reply:
IDK, I think we do not allow custom strings

Reply:
I also think we don't allow it